it/src/main/java/com/google/cloud/teleport/it/common/JDBCBaseIT.java

```diff
@@ -0,0 +1,146 @@
+  // NOTE: These versions must correspond to the versions declared in the `it/pom.xml` file.
+  private static final String MYSQL_VERSION = "8.0.30";
+  private static final String POSTGRES_VERSION = "42.2.27";
+  private static final String ORACLE_VERSION = "19.3.0.0";
+  private static final String MSSQL_VERSION = "6.4.0.jre8";
+
+  @Before
+  public void setUpJDBC() throws IOException {
+
+    String basePath = getMvnBaseRepoPath();
+
+    String mySqlDriverGCSRelativePath = GCS_PREFIX + mySqlDriverLocalJar();
+    String postgresDriverGCSRelativePath = GCS_PREFIX + postgresDriverLocalJar();
+    String oracleDriverGCSRelativePath = GCS_PREFIX + oracleDriverLocalJar();
+    String msSqlDriverGCSRelativePath = GCS_PREFIX + msSqlDriverLocalJar();
+
+    gcsClient.uploadArtifact(mySqlDriverGCSRelativePath, mySqlDriverLocalPath(basePath));
+    gcsClient.uploadArtifact(postgresDriverGCSRelativePath, postgresDriverLocalPath(basePath));
+    gcsClient.uploadArtifact(oracleDriverGCSRelativePath, oracleDriverLocalPath(basePath));
+    gcsClient.uploadArtifact(msSqlDriverGCSRelativePath, msSqlDriverLocalPath(basePath));
+  }
+
+  protected String mySqlDriverGCSPath() {
+    return getGcsPath(GCS_PREFIX + mySqlDriverLocalJar());
+  }
+
+  protected String postgresDriverGCSPath() {
+    return getGcsPath(GCS_PREFIX + postgresDriverLocalJar());
+  }
+
+  protected String oracleDriverGCSPath() {
+    return getGcsPath(GCS_PREFIX + oracleDriverLocalJar());
+  }
+
+  protected String msSqlDriverGCSPath() {
+    return getGcsPath(GCS_PREFIX + msSqlDriverLocalJar());
+  }
+
+  private static String mySqlDriverLocalJar() {
+    return String.join("-", MYSQL_JAR_PREFIX, MYSQL_VERSION) + JAR_SUFFIX;
+  }
+
+  private static String postgresDriverLocalJar() {
+    return String.join("-", POSTGRES_JAR_PREFIX, POSTGRES_VERSION) + JAR_SUFFIX;
+  }
+
+  private static String oracleDriverLocalJar() {
+    return String.join("-", ORACLE_JAR_PREFIX, ORACLE_VERSION) + JAR_SUFFIX;
+  }
+
+  private static String msSqlDriverLocalJar() {
+    return String.join("-", MSSQL_JAR_PREFIX, MSSQL_VERSION) + JAR_SUFFIX;
+  }
+
+  private static String mySqlDriverLocalPath(String basePath) {
+    return String.join("/", basePath, MYSQL_LOCAL_PATH, MYSQL_VERSION, mySqlDriverLocalJar());
+  }
+
+  private static String postgresDriverLocalPath(String basePath) {
+    return String.join(
+        "/", basePath, POSTGRES_LOCAL_PATH, POSTGRES_VERSION, postgresDriverLocalJar());
+  }
+
+  private static String oracleDriverLocalPath(String basePath) {
+    return String.join("/", basePath, ORACLE_LOCAL_PATH, ORACLE_VERSION, oracleDriverLocalJar());
+  }
+
+  private static String msSqlDriverLocalPath(String basePath) {
+    return String.join("/", basePath, MSSQL_LOCAL_PATH, MSSQL_VERSION, msSqlDriverLocalJar());
+  }
+
+  private String getMvnBaseRepoPath() {
+    String mavenCmd = "mvn help:evaluate -Dexpression=settings.localRepository -q -DforceStdout";
```

Comment:
Don't have a concrete example but I think this may be flaky / environment based. For example, when I run CI jobs (Jenkins) I sometimes set the job to use a private repository for the run (so concurrent builds don't interact with each other) and I don't know how this will affect that.

Maybe we can allow the path to be sent through -DmavenRepository=, and use this is missing. (Just wanted to point that out, I don't think it's strictly necessary at this point)

Reply:
Fixed.